pytest_plugins/metadata_markers.py

```diff
@@ -164,7 +164,7 @@ def pytest_collection_modifyitems(items, config):
     # split the option string and handle no option, single option, multiple
     # config.getoption(default) doesn't work like you think it does, hence or ''
     importance = [i for i in (config.getoption('importance') or '').split(',') if i != '']
```

Comment:
Mind doing the same to importance, while at it?
```suggestion
    importance = [i.lower() for i in (config.getoption('importance') or '').split(',') if i != '']
```
+ update the commit message